features/cli/create.feature

```diff
@@ -167,3 +167,32 @@ Feature: creating 'apps' with CLI
     Then the step should fail
     And the output should contain "No Dockerfile"
 
+  # @author chuyu@redhat.com
+  # @case_id OCP-22515
+  Scenario: 4.x Could not create any context in non-existent project
+    Given I create a new application with:
+      | docker image | openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world |
+      | name         | myapp          |
+      | n            | noproject      |
+    Then the step should fail
+    Then the output should match "User "<%=@user.name%>" cannot create resource "imagestreamimports".*in the namespace "noproject""
+    Given I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/pods/hello-pod.json |
+      | n | noproject |
+    Then the step should fail
+    Then the output should match "User "<%=@user.name%>" cannot create resource "pods".*in the namespace "noproject""
+    Given I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/deployment/deployment1.json |
+      | n | noproject |
+    Then the step should fail
+    Then the output should match "User "<%=@user.name%>" cannot create resource "deploymentconfigs".* in the namespace "noproject""
+    Given I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift/origin/master/examples/image-streams/image-streams-centos7.json |
+      | n | noproject |
+    Then the step should fail
+    Then the output should match "User "<%=@user.name%>" cannot create resource "imagestreams".* in the namespace "noproject""
+    Given I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift/origin/master/examples/sample-app/application-template-stibuild.json |
+      | n | noproject |
+    Then the step should fail
+    Then the output should match "User "<%=@user.name%>" cannot create resource "templates".* in the namespace "noproject""
```

Comment:
all these tests can happily be unit tests. This will be better to avoid breakage due to human readable strings in the future. Otherwise looks good and can merge if you wish them here.

Reply:
Yes, should be removed.